utide/_reconstruct.py

```diff
@@ -20,15 +20,15 @@ def reconstruct(
     Parameters
     ----------
     t : array_like
-        Time in days since ``epoch``.
+        Time in days since ``epoch``, or array of datetime, np.datetime64, or pd.datetime
     coef : `Bunch`
         Data structure returned by `utide.solve`.
-    epoch : {string, `datetime.date`, `datetime.datetime`}, optional
-        Valid strings are 'python' (default); 'matlab' if `t` is
-        an array of Matlab datenums; or an arbitrary date in the
-        form 'YYYY-MM-DD'.  The default corresponds to the Python
-        standard library `datetime` proleptic Gregorian calendar,
-        starting with 1 on January 1 of year 1.
+    epoch : {string, `datetime.date`, `datetime.datetime`}, if datenum is provided in t.
+        Default `None` if `t` is `datetime`, `np.datetime64`, or `pd.datetime array.`
+        Optional valid strings are
```

Comment:
Maybe add one more: the unix epoch, since that is the new mpl datenum default epoch, as well as the epoch for np.datetime64.  I think it would be straightforward to make it the internal epoch as well.

Reply:
Yes, ok for adding `unix` epoch.

Regarding internal epoch - please see my query in the utide/_time_conversion.py comment above.